crates/cxx-qt-gen/src/parser/invokable.rs

```diff
@@ -42,18 +42,27 @@ pub struct ParsedQInvokable {
     pub parameters: Vec<ParsedFunctionParameter>,
     /// Any specifiers that declared on the invokable
     pub specifiers: HashSet<ParsedQInvokableSpecifiers>,
+    /// Whether the method is qinvokable
+    pub is_qinvokable: bool,
 }
```

Comment:
This naming is now confusing. A struct called `ParsedQInvokable` shouldn't be able to maybe not be a QINVOKABLE. Can you come up with a new name for this? Maybe `ParsedMethod` or something?

Reply:
Done renamed most of the invokable naming to method :-)